content/news/2023-10-13-EGD-2023-the-wrapup/index.md

```diff
@@ -0,0 +1,141 @@
+
+Speaking of funding we like to emphasize ELIXIR's ([again!](/events/2022-10-egd/#supporters)) great role in enabling us again to carry out the EGD. We thank both Hub and DE node for their generous funding of the _overall_ event and supportive way in organizing!
+
+### EuroScienceGateway's first General Assembly (GA) 
+
+The sub-meeting mostly set on Day 3 in parallel with the CoFest marked the project's turnover from year 1 to year 2, and entering the finalization phase for our first report (which is why we had a dedicated BoF). Thus, we had a wrap-up of our achievements, and could note that we are well on track having ESG's proposal in mind. 
+
+In a nutshell: 
+ * usegalaxy.eu's user base is growing fast, having passed 77k recently
+ * ESG is facilitating **deeper integrations within EOSC**, e.g. via the [Macro-Roadmap](https://eosc.eu/roadmap), agreements with [FAIR-EASE](https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces) (plus a [workshop](/news/2023-05-21-fair-ease-euro-science-gateway/)) and [OpenAIRE](/news/2023-05-05-eosc-esg-openaire/)
+ * [Exporting **RO-Crate and BCO** from Galaxy](https://galaxyproject.org/news/2023-02-23-structured-data-exports-ro-bco/) and [**publishing** from Galaxy via Invenio to Zenodo](https://github.com/galaxyproject/galaxy/pull/16381) ([WP2: Stimulate FAIR and reusable research](https://galaxyproject.org/news/2023-01-12-eurosciencegateway-wp2/))
+ * Update and extension of the **Pulsar network**; [new repository available](https://github.com/usegalaxy-eu/pulsar-deployment) and [development of a GA4GH TES-compliant API](https://github.com/CESNET/tesp-api) ([WP3: Distributed heterogeneous compute](https://galaxyproject.org/news/2023-01-19-eurosciencegateway-wp3/))
+ * Milestone on **Bring Your Own Compute/Storage** ([BYOC](https://drive.google.com/file/d/1I3FCJ8vmo6jESNqMcfGZHRcVxtk169Tm/view)/[BYOS](https://drive.google.com/file/d/1IUBbuowoCYsOAuaKzBtEtwayzY4oooQF/view)) achieved; principle adoption of [ARC as job runner](https://docs.google.com/presentation/d/1xb70Z4dZfXEJrvbXFhLQceA1g2UZnEWG) and efforts on [meta-scheduling](https://docs.google.com/presentation/d/1-5ch2UpOXH9qBouR6cbqBYbsgzSNMysk/edit#slide=id.p1)([WP4: Building blocks for a sustainable operating model](https://galaxyproject.org/news/2022-12-08-esg-wp4/))
```

Comment:
why mentioning this Milestone? "Milestone on **Bring Your Own Compute/Storage** " - there were more deliverables an milestones if you want to mention them here